lms/static/sass/elements/_program-card.scss

```diff
@@ -67,39 +67,47 @@
     @include susy-media($bp-screen-lg) { margin-top: 135px; }
   }
 
-  .meta-info{
+  .meta-info {
     font-size: font-size(x-small);
     color: palette(grayscale, dark);
     position: absolute;
     top: $baseline;
     width: calc(100% - 40px);
   }
 
-  .organization{
+  .organization {
     @include span(6);
     white-space: nowrap;
     overflow: hidden;
   }
 
-  .category{
+  .category {
     @include span(6);
     @include text-align(right);
 
-    .category-text{
-        @include float(right);
+    .category-text {
+      @include float(right);
     }
 
-    .xseries-icon{
+    .category-icon {
       @include float(right);
       @include margin-right($baseline*0.25);
-      background: url('#{$static-path}/images/icon-sm-xseries-black.png') no-repeat;
       background-color: transparent;
       background-size: 100%;
       width: ($baseline*0.7);
       height: ($baseline*0.7);
     }
+
```

Comment:
In this case, there's no need to adjust SVG properties like fill and stroke. Including the SVG inline would have been heavy-handed.